### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by ajeddeloh**
Minor quality of life fixes: don't compile things we don't need to and log Ignition's PID when running in a BB test so you can look at the logs more easily.

Depends on https://github.com/coreos/ignition/pull/481

### CODE REVIEW

**Comment by ajeddeloh** [comment]
looks like tailor really doesn't like `=` in it's regexps. I also tried escaping it like `\=` but that didn't help. 

cc @crawford

**Comment by ajeddeloh** [comment]
Added a bunch more stuff because I kept finding things broken in the tests/build scripts/travis config. PTAL again.

File: build [hunk 1]
```diff
@@ -22,9 +22,13 @@ export CGO_ENABLED=1
...
 
-for D in tests/stubs/*; do
-	if [ -d "${D}" ]; then
-		echo "Building ${D}"
-		go install ${REPO_PATH}/${D}
-	fi
-done
+if [ "$GOHOSTARCH" == "$GOARCH" ]; then
+	# Don't try to go install things when cross compiling because
+	# go install disallows it.
```
Comment (dgonyeo) [inline build]: Would it make more sense to run `go build` when cross compiling, to at least check that these binaries compile?

**Comment by ajeddeloh** [comment]
tailor disable commit title

**Comment by ajeddeloh** [comment]
Removing the tailor commit since it's broken. Switched to using `go build` when `go install` won't work

**Pull request merged**